<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yakunin</class>
 <widget class="QMainWindow" name="yakunin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>234</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>yakunin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(14, 73, 85);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_photo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 225, 225);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>фото</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_date_rod">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>176</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Дата рождения</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_m">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>29</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>м</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_load">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(6, 34, 40);
background-color: rgb(15, 81, 94);
color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ready">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(6, 34, 40);
background-color: rgb(15, 81, 94);
color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Готов</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>298</x>
      <y>176</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(6, 34, 40);
background-color: rgb(15, 81, 94);
color: rgb(225, 225, 225);</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_j">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>ж</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dlc">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(6, 34, 40);
background-color: rgb(15, 81, 94);
color: rgb(225, 225, 225);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fio">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>23</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dolg">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>57</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Должность</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stag">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Стаж работы</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_stag">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>58</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(6, 34, 40);
background-color: rgb(15, 81, 94);
color: rgb(225, 225, 225);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dolg">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>93</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(6, 34, 40);
background-color: rgb(15, 81, 94);
color: rgb(225, 225, 225);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fio">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>23</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(6, 34, 40);
background-color: rgb(15, 81, 94);
color: rgb(225, 225, 225);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
